### PR TITLE
refactor(frontend): move ContextMenu as More button among Hero buttons

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We move the `ContextMenu` component among the Hero buttons.

Screenshot 2024-07-23 at 13 19 05
Screenshot 2024-07-23 at 13 19 24

Screenshot 2024-07-23 at 13 19 40
Screenshot 2024-07-23 at 13 20 03

### CODE REVIEW

File: src/frontend/src/lib/components/hero/HeroContent.svelte [hunk 1]
```diff
@@ -55,8 +53,11 @@
 {/if}
 
 {#if actions}
-	<div transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}>
-		<Actions {send} />
+	<div
+		transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}
+		class="flex w-full justify-center"
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/hero/HeroContent.svelte:58]: Adding the class preserves the smoothness of the animation, right?

**Review by peterpeterparker — COMMENTED** [review]
On mobile devices when "Convert" button is displayed, is there enough space for all four buttons?

**Comment by AntonioVentilii-DFINITY** [comment]
> On mobile devices when "Convert" button is displayed, is there enough space for all four buttons?


Yes, I put class justify-between to adapt them, see below:

Screenshot 2024-07-23 at 15 11 59

File: src/frontend/src/lib/components/hero/HeroContent.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/HeroContent.svelte:58]: correct! it should not interfere. I slowed down the transition to test it:


https://github.com/user-attachments/assets/c72f7391-2e0b-47e1-82dc-4210e9e86d8c

Comment (peterpeterparker) [reply src/frontend/src/lib/components/hero/HeroContent.svelte:58]: Thanks!

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks

**Pull request merged**